index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Portfolio Website</title>
+
+ <!--icon link-->
+ <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.0/css/line.css">
+ <script src="https://kit.fontawesome.com/9ea65bb5f9.js" crossorigin="anonymous"></script>
+
+ <!--css link-->
+ <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+  <div class="wrapper">
+    <!--header section-->
+    <header class="header">
+
+      <!--logo-->
+      <a href="#index.html" class="logo">My logo</a>
+
+      <!--navbar-->
+      <nav>
+        <ul class="navlink">
+          <li><a href="#index.html">Portfolio</a></li>
+          <li><a href="#about.html">About</a></li>
+          <li><a href="#contact.html">Contact</a></li>
+        </ul>
+      </nav>
+      <!--end of navbar-->
+
+      <div class="toggle">
+        <i class="uil uil-bars"></i>
+      </div>
+      
+    </header>
+
+    <!--home page-->
+    <section class="home">
+
+      <article class="home-content">
+      <h1 class="home-title">I'm Alexandra <br> Glad to see you!</h1>
+
+      <p class="home-main">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+      <span>LET'S CONNECT</span>
+
```

Comment:
- [ ] I would suggest you please use `hx` tag for the sub-title for better semantic HTML